Library/Homebrew/resource_auditor.rb

```diff
@@ -100,6 +100,15 @@ def self.curl_deps
       end
     end
 
+    def audit_resource_name_matches_pypi_package_name_in_url
+      return unless url.match?(%r{^https?://files\.pythonhosted\.org/packages/})
+
+      pypi_package_name = url.split("/").last.split(/-\d+\.\d+./).first.tr("_", "-")
```

Comment:
Here's other characters of interest: https://github.com/Homebrew/brew/blob/e7d0625776dca58c81b03c19eeda1af8bd6bb58e/Library/Homebrew/utils/pypi.rb#L345-L349